src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/typename/Example1.java

```diff
@@ -0,0 +1,18 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="TypeName"/>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.naming.typename;
+
+// xdoc section -- start
+class Example1{
```

Comment:
```suggestion
class Example1 {
```

Please do this for all examples.

Reply:
done